.github/workflows/build-windows.yml

```
@@ -39,8 +38,8 @@ jobs:
     - name: Install uv
       uses: ./.github/actions/setup-uv
 
-    - name: Install sccache
-      run: choco install -y sccache
+    - name: Setup sccache
+      uses: Mozilla-Actions/sccache-action@7d986dd989559c6ecdb630a3fd2557667be217ad # v0.0.9
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)